test/python/circuit/test_scheduled_circuit.py

```diff
@@ -50,21 +51,24 @@ def setUp(self):
     def test_schedule_circuit_when_backend_tells_dt(self):
         """dt is known to transpiler by backend"""
         qc = QuantumCircuit(2)
-        qc.delay(0.1, 0, unit="ms")  # 450000[dt]
+        qc.delay(0.1, 0, unit="ms")  # 450450[dt]
         qc.delay(100, 0, unit="ns")  # 450[dt]
-        qc.h(0)  # 160[dt]
-        qc.h(1)  # 160[dt]
-        sc = transpile(qc, self.backend_with_dt, scheduling_method="alap", layout_method="trivial")
-        self.assertEqual(sc.duration, 450546)
+        qc.h(0)  # 195[dt]
+        qc.h(1)  # 210[dt]
+
+        backend = GenericBackendV2(27, calibrate_instructions=True, seed=42)
```

Comment:
I am not 100% sure it will not trigger changes, but in case you want to try:

```suggestion
        backend = GenericBackendV2(2, calibrate_instructions=True, seed=42)
```

Reply:
yeap.. 2 works